content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -76,6 +77,30 @@ samplingRatePerMillion: 100
 For more information about the `TracingConfiguration` struct, see
 [API server config API (v1alpha1)](/docs/reference/config-api/apiserver-config.v1alpha1/#apiserver-k8s-io-v1alpha1-TracingConfiguration).
 
+### kubelet traces
+
```

Comment:
We could put a specific feature state shortcode here.

Reply:
updated